src/components/Books.js

```diff
@@ -4,20 +4,64 @@ import store from '../redux/configureStore';
 
 const ADD_BOOK = { type: 'ADD_BOOK', title: '', category: '' };
 const REMOVE_BOOK = { type: 'REMOVE_BOOK', id: 0 };
+const FETCH_BOOK = { type: 'FETCH_BOOK', payload: [] };
 
-const BookList = () => {
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/kHXXub8OPr6PqC5APwrM/books';
+
+const fetchData = async () => {
+  fetch(baseUrl, { method: 'GET' })
+    .then((response) => response.json()
+      .then((data) => store.dispatch({ ...FETCH_BOOK, payload: { data } })));
+};
+
+fetchData();
```

Comment:
- [ ] It would be better if we make use of the react hooks (useEffect) hook. With this hook, our fetch data function gets called when the component is mounted in the DOM.

Reply:
Done! Thanks for the comments!